### ISSUE TITLE
Refactoring bug when running "Add accessibility modifier" to extern methods

### ISSUE THREAD

**Issue by IEvangelist** [dotnet/roslyn#44145]
**Version Used**: 

Microsoft Visual Studio Enterprise 2019
Version 16.5.4
VisualStudio.16.Release/16.5.4+30011.22
Microsoft .NET Framework
Version 4.8.03752
C# Tools   3.5.0-beta4-20153-05+20b9af913f1b8ce0a62f72bea9e75e4aa3cf6b0e

**Steps to Reproduce**:

1. Create a new .NET Core console project, paste in the following C# code:
  ```csharp
  using System.Runtime.InteropServices;
  using Microsoft.Win32.SafeHandles;
  
  class Program
  {
      [DllImport("kernel32.dll")]
      static extern int GetFileSize(SafeFileHandle hFile, out int lpFileSizeHigh);
  }
  ```
2. Place cursor just before the `static` keyword, and press <kbd>Ctrl+.</kbd>  (or open **Quick Actions and Refactorings...**, then select **Add accessibility modifier** option to address **IDE0040 Accessibility modifiers required**

Even observing the generated preview of the difference highlights the bug.

![image](https://user-images.githubusercontent.com/7679720/81594070-f8a94e00-9385-11ea-9446-45d99ea1ae99.png)

**Expected Behavior**:

I would expect that the refactoring would only add the `private` access modifier, and not remove the `extern` keyword. After applying the refactoring I expect the code to be this:

  ```csharp
  using System.Runtime.InteropServices;
  using Microsoft.Win32.SafeHandles;
  
  class Program
  {
      [DllImport("kernel32.dll")]
      private static extern int GetFileSize(SafeFileHandle hFile, out int lpFileSizeHigh);
  }
  ```

**Actual Behavior**:

The `extern` keyword is removed, thus breaking the code. After applying the refactoring this is the resulting broken code:

  ```csharp
  using System.Runtime.InteropServices;
  using Microsoft.Win32.SafeHandles;
  
  class Program
  {
      [DllImport("kernel32.dll")]
      private static int GetFileSize(SafeFileHandle hFile, out int lpFileSizeHigh);
  }
  ```

Resulting in two errors:

| Severity | Code | Description |
|:-|:-|:-|
| Error | [CS0601](https://www.bing.com/search?q=CS0601+C%23+The+DllImport+attribute+must+be+specified+on+a+method+marked+%27static%27+and&form=VSHELP) | The `DllImport` attribute must be specified on a method marked `static` and `extern` |
| Error | [CS0501](https://www.bing.com/search?q=CS0501+C%23+must+declare+a+body+because+it+is+not+marked+abstract%2c+extern%2c+or+partial&form=VSHELP) | `Program.GetFileSize(SafeFileHandle, out int)` must declare a body because it is not marked abstract, extern, or partial |

**Comment by sharwell** [comment]
Duplicate of #29561

**Comment by sharwell** [comment]
@IEvangelist this should be fixed in 16.6 Preview 2 and newer 👍
